src/Expression/LinkExpressionFunction.php

```diff
@@ -15,13 +15,14 @@ class LinkExpressionFunction implements ExpressionFunctionProviderInterface
      * @phpcsSuppress SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingReturnTypeHint
      * @phpcsSuppress SlevomatCodingStandard.TypeHints.TypeHintDeclaration.UselessReturnAnnotation
      */
-    public function getFunctions()
+    public function getFunctions(): array
```

Comment:
is this a bc break? is PHP going to complain if someone extended this class and declared a method without the return type hint?

Reply:
I have reverted that change